src/fosslight_source/_scan_item.py

```diff
@@ -84,17 +84,21 @@ def set_fileURL(self, value):
     def set_vendor(self, value):
         self.vendor = value
 
-    def get_row_to_print(self):
-        if not self.download_location:
+    def get_row_to_print(self, column_number=-1):
+        if not column_number > 0:
```

Comment:
My suggestion : 
`if not column_number > 0:` -> `if column_number <= 0`

Reply:
I'll separate the get_row_to_print method in to two. One that include scancoss result and one that doesn't.